libs/ledger-live-common/src/apps/support.ts

```diff
@@ -35,6 +35,7 @@ const appVersionsRequired = {
   "Cardano ADA": ">= 4.1.0",
   Zcash: "> 2.0.6",
   NEAR: ">= 1.2.1",
+  "Tezos Wallet": ">= 2.4.3",
```

Comment:
@lvndry In the end the app that landed on P1 is **2.4.5**, so let's change this just to be safe